features.org

```diff
@@ -305,19 +299,44 @@ Tutorial on [[https://orgmode.org/worg/org-tutorials/org-custom-agenda-commands.
 
 ** Description
 
-Adding TODO items to your =.org= files is called /capturing/.
+You can add content to an =.org= file by /capturing/ data.
 
-You can capture from everywhere, both within Emacs and from other
-applications like your web browser, PDF viewer, etc.
+You can capture from anywhere -- from an Emacs buffer or from other
+applications like your web browser, PDF viewer, and more.
 
-Capture templates provide a quick and effective way to save information to look
-at later.
+Capture templates provide a quick and effective way to ingest
+structured data for a variety of workflows, from adding TODO items to
+quoting the relevant section of a manual or saving a receipt.
 
 *** More
 
 More on [[file:manual/Capture.html][capturing]] (manual)
+* Emacs Extensible
```

Comment:
I kind of prefer "Extremely Extensible" or similar, I wonder if this may slightly confuse non-Emacsers looking at this page.

Reply:
I changed the headline, but kept the Emacs version in the text. Let's see what the general take on it is and we can adjust if needs be.

Reply:
I feel this is an approvement. Let's leave this be for now.